types/xml-c14n/index.d.ts

```diff
@@ -0,0 +1,31 @@
+// Type definitions for xml-c14n 0.0
+// Project: https://github.com/deoxxa/xml-c14n
+// Definitions by: Konstantin Yuriev <https://github.com/gallowsmaker>
+//                 Max Boguslavskiy <https://github.com/maxbogus>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.7
+
+declare namespace xml_c14n {
```

Comment:
If you want to be able to import these types, the name of the namespace and function need to match.

Reply:
reworked this issue. please check.